lib/new_relic/agent/opentelemetry/trace/tracer.rb

```diff
@@ -27,6 +27,7 @@ def start_span(name, with_parent: nil, attributes: nil, links: nil, start_timest
 
             span = get_span_from_finishable(finishable)
             span.finishable = finishable
+            span.add_attributes(attributes) if attributes
```

Comment:
I'm not 100% sure about this, but it seems if a span is create with attributes, they should be added

Reply:
I think this is a good call! My only recommendation would be to add some tests to the `tracer_test.rb` file that make sure the attributes get assigned. I don't think this will be hit by any of the cross-agent tests.